proto/astria/sequencer/v1alpha1/block.proto

```diff
@@ -41,3 +41,15 @@ message SequencerBlock {
   // the rollup transactions.
   astria.sequencer.v1alpha1.Proof rollup_ids_proof = 4;
 }
+
+// `RollupSpecificSequencerBlock` is a subset of `SequencerBlock` that only contains
+// transaction data for a specific rollup.
+message RollupSpecificSequencerBlock {
```

Comment:
Should this be flattened or contain the `RollupTransactions` object?

Reply:
this needs the proofs + commitments also i believe? the conductor will be calling `GetRollupSpecificSequencerBlock` if i understand correctly, so needs to verify it

Reply:
also not a huge fan of the name, maybe `RollupDataSubset` or something is better?

Reply:
If you get the data directly from a trusted sequencer node what would be verified?

I guess if we added a light client to conductor you could verify from a sequencer node against header?

Reply:
```
    rollup_transactions_proof: merkle::Proof,
    rollup_ids_proof: merkle::Proof,
```

see `SequencerBlock` in astria-core, you would just need to add these. and yep I'm thinking in the case that the sequencer node is not trusted and you're just getting the data for light client verification

Reply:
Should we just eliminate this structure altogether? We could use the same SequencerBlock type, but filtered down. This would allow us to expand the API in the future and say request the transactions for multiple rollups.